src/idiom327ex.fz

```diff
@@ -0,0 +1,3 @@
+convert_to_lower_case is
```

Comment:
The outer feature should be named after the idiom number, like in the other idioms

```suggestion
ex327 is
```